docker/misk-build/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM squareup/misk-node:0.0.1
+
+COPY misk-build.sh /usr/local/bin/
+RUN ln -s usr/local/bin/misk-build.sh
+ENTRYPOINT ["misk-build.sh"]
```

Comment:
Think you want CMD here although in the case it may not matter too much:
http://goinbigdata.com/docker-run-vs-cmd-vs-entrypoint/

The default entry point would be bash and the command would be misk-build.

Reply:
I'd just merge these two docker files into one unless there's something I'm missing? Do you not want that misk-build.sh in the base for some reason.

Reply:
We have another planned image to spin up dev servers so we may keep them as separate docker files for now.